usecases/auth/authorization/rbac/types.go

```diff
@@ -52,10 +52,21 @@ func policy(permission *models.Permission) (resource, verb, domain string) {
 	case cluster:
 		resource = authorization.Cluster()
 	case collections:
+		if permission.Collection == nil {
+			return
```

Comment:
**issue:** that should never happen `permission.Collection` default is "*" , we need to check against and make the default for resource '*'